Userland/Applications/SpaceAnalyzer/main.cpp

```diff
@@ -259,7 +259,8 @@ static void analyze(RefPtr<Tree> tree, SpaceAnalyzer::TreeMapWidget& treemapwidg
             if (!first) {
                 builder.append(", ");
             }
-            builder.append(strerror(key));
+            auto const* error = strerror(key);
+            builder.append({ error, strlen(error) });
```

Comment:
Feels like an unfortunate side-effect of this change. Maybe it'd still be good to have a `builder.append(char const*)` API? Not sure, it could also be left as-is.